current-work/README.md

```diff
@@ -0,0 +1,48 @@
+# The JSON Schema Organization's areas of focus or interest
+
+## About
+### What is this space?
+
+During an [Open Community Working Meeting](https://github.com/json-schema-org/community/issues/244), it was suggested that we could have a place for the core team (and maybe active community members?) to log their current areas of focus or interest in the form of GitHub Issues, Pull Requests, and Discussions.
+
+### Why?
+
+We concluded there was a LOT of activity taking place, and it was hard to get people to focus on the "important things". It is indeed hard to focus on priority items when there are many new GitHub Issues and Discussions each week, especially for those who are community members or not working on JSON Schema full or even part time. It was suggested that by asking for one to three items of priority, we might find it easier to draw attention where we feel it is needed.
```

Comment:
I'd change 'focus on the "important things"' to 'respond to the most immediately important things' or something of that sort.  This is about responding to each other, rather than priorities in general.